UnitsNet.Tests/GeneratedCode/TestsBase/AccelerationTestsBase.g.cs

```diff
@@ -118,25 +118,6 @@ public void DefaultCtor_ReturnsQuantityWithZeroValueAndBaseUnit()
             Assert.Equal(AccelerationUnit.MeterPerSecondSquared, quantity.Unit);
         }
 
-        [Fact]
-        public void Ctor_WithInfinityValue_ThrowsArgumentException()
```

Comment:
Shouldn't we change the tests to reflect that Nan/Inf are allowed, instead of removing the tests?

Reply:
Make sense. Will do that.

Reply:
Finally, finally! made the changes.